lib/src/compiler/output/abstract_emitter.dart

```diff
@@ -321,6 +321,16 @@ abstract class AbstractEmitterVisitor
     return null;
   }
 
+  dynamic visitIfNullExpr(o.IfNullExpr ast, dynamic context) {
+    EmitterVisitorContext ctx = context;
+    ctx.print('''(''');
```

Comment:
Using triple quotes here serves no purpose. Single quotes will do, and can be used here (Line 326), as well as lines 328 and 330.